deployments/ansible/group_vars/signalfx-host-group

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think this file should be in the `defaults` dir.  Look at how the [MongoDB role](https://github.com/William-Yeh/ansible-mongodb) and [Nginx role](https://github.com/geerlingguy/ansible-role-nginx) do it.

Reply:
The existing structure set variables via facts according to the values defined per inventory group or all, followed the official ansible structure [MongoDB] (https://github.com/ansible/ansible-examples/tree/master/mongodb) and [other examples] (https://github.com/ansible/ansible-examples) which overrides the defaults one. Yes, the `defaults` dir from above example make more sense in defining default values for *signalfx-agent* role. Will update the structure accordingly!